cf-editor-rte-toolbar-button-sample/src/aem-cf-editor-1/web-src/src/index.js

```diff
@@ -0,0 +1,73 @@
+/* 
+* <license header>
+*/
+
+import 'core-js/stable'
+import 'regenerator-runtime/runtime'
+
+window.React = require('react')
+import ReactDOM from 'react-dom'
+
+import Runtime, { init } from '@adobe/exc-app'
+
+import App from './components/App'
+import './index.css'
+/* Here you can bootstrap your application and configure the integration with the Adobe Experience Cloud Shell */
+try {
+  // attempt to load the Experience Cloud Runtime
+  require('./exc-runtime')
+  // if there are no errors, bootstrap the app in the Experience Cloud Shell
+  init(bootstrapInExcShell)
+} catch (e) {
+  console.log('application not running in Adobe Experience Cloud Shell')
+  // fallback mode, run the application without the Experience Cloud Runtime
+  bootstrapRaw()
+}
+
+function bootstrapRaw () {
+  /* **here you can mock the exc runtime and ims objects** */
+  const mockRuntime = { on: () => {} }
+  const mockIms = {}
+
+  // render the actual react application and pass along the runtime object to make it available to the App
+  ReactDOM.render(
+    <App runtime={mockRuntime} ims={mockIms} />,
+    document.getElementById('root')
+  )
+}
+
+function bootstrapInExcShell () {
```

Comment:
Do we need this method?

Reply:
Please look at [this comment](https://github.com/adobe/aem-uix-examples/pull/7#issuecomment-1863455698)